glide-core/redis-rs/redis/src/cluster_async/mod.rs

```diff
@@ -495,14 +496,27 @@ pub(crate) struct ClusterConnInner<C> {
 
 impl<C> Dispose for ClusterConnInner<C> {
     fn dispose(self) {
+        if let Ok(conn_lock) = self.inner.conn_lock.try_read() {
+            // Each node may contain user and *maybe* a management connection
```

Comment:
minor: I think this commend can be removed now that the count is being done in connections_count